Include/Acidanthera/Library/OcDevicePathLib.h

```diff
@@ -377,4 +377,11 @@ OcGetNextLoadOptionDevicePath (
   IN EFI_DEVICE_PATH_PROTOCOL  *FullPath
   );
 
+BOOLEAN
+OcDevicePathHasFilePathSuffix (
+  IN EFI_DEVICE_PATH_PROTOCOL  *DevicePath,
+  IN CHAR16                    *Suffix,
```

Comment:
- Needs docs?
- Why not CONST for the Suffix?

Reply:
Because the subfunctions like OcStriCmp do not take CONST, could drop another commit at some point.